docs/field-level-encryption.md

```diff
@@ -151,3 +151,68 @@ To declare a field as encrypted, you must:
 2. Choose an encryption type for the schema and configure the schema for the encryption type
 
 Not all schematypes are supported for CSFLE and QE.  For an overview of valid schema types, refer to MongoDB's documentation.
+
+### Registering Models
+
+Encrypted schemas must be registered on a connection, not the Mongoose global:
```

Comment:
Is there a particular reason for this? `mongoose.model('User', encryptedUserSchema)` is equivalent to `mongoose.connection.model('User', encryptedUserSchema)` so there shouldn't be any reason why you can't register an encrypted model on the Mongoose global. Unless there's some required configuration for `createConnection()`.

Also, should be "Encrypted models must be registered on a connection"

Reply:
The requirements for CSFLE are:

- auto encryption options must be provided to the mongoose connection creation method
- schemas and models must be declared before mongoose instantiates a mongo client

The workflow I imagined was:

```javascript
const connection = createConnection();
// declare encrypted schemas, initialize models on `connection`
await connection.openUri(<>, { autoEncryption: { ... } });
```

I guess this would also work with the global connection, like so?

```
// declare encrypted schemas, initialize models on the mongoose global
const connection = mongoose.connect(<>, { autoEncryption: { ... } });
```

Do you see any issues with this?  If not, I'll add tests just to confirm and update the docs.

Reply:
Yup, this works!  I added a test for this scenario.